lua/textcase/plugin/conversion.lua

```diff
@@ -95,11 +92,21 @@ function M.do_lsp_rename(method)
     local params = lsp.util.make_position_params()
     params.newName = method(current_word)
 
-    if flag_buf_request_all then
-      -- lsp.buf_request_all(0, "textDocument/rename", params)
-    else
-      lsp.buf_request(0, "textDocument/rename", params)
-    end
+    lsp.buf_request_all(0, "textDocument/rename", params, function(results)
+      local total_files = 0
+      for client_id, response in pairs(results) do
+        if not response.error then
+          local client = vim.lsp.get_client_by_id(client_id)
+          vim.lsp.util.apply_workspace_edit(response.result, client.offset_encoding)
+
+          -- after the edits are applied, the files are not saved automatically.
+          -- let's remind ourselves to save those...
+          -- TODO: This will be modified to include only one of the clients count
+          total_files = vim.tbl_count(response.result.changes)
+        end
+      end
+      print(string.format("Changed %s file%s. To save them run ':wa'", total_files, total_files > 1 and "s" or ""))
```

Comment:
Maybe we can only print something if it changed more than one file? For one file, I wouldn't care if it's saved or not as I'm working on that specific file anyways right now.

Also, a prefix might be helpful e.g. "Textcase: Changed ..."

Reply:
Now that I think about it again, even a message for a single would be interesting because it gives feedback that the LSP rename worked.

Reply:
That's the reason for [this comment](https://github.com/johmsalas/text-case.nvim/blob/70da1d5d76587f0f139a22d63e263f8ddac65659/lua/textcase/plugin/conversion.lua#L104).

[There is currently a bug](https://github.com/johmsalas/text-case.nvim/issues/107) when multiple clients do `textDocument/rename`, the solution for this will be, given the nature of `textDocument/rename` it should only apply one of the clients, not all of them, I'll add some logic to chose one of the clients in the next PR

I didn't include it in this PR to separate the deliveries into small chunks, but in fact, I wouldn't have prioritized buf_request_all into the first milestone if it wasn't part of the solution to the multi LSP clients issue

Reply:
Alright, thanks for the explanation :)

I would really like to dive into #107